loader/src/main/java/net/neoforged/fml/loading/ModDirTransformerDiscoverer.java

```diff
@@ -82,7 +84,7 @@ private static void scan(final Path gameDirectory) {
             return;
         }
         try (var walk = Files.walk(modsDir, 1)){
-            walk.forEach(ModDirTransformerDiscoverer::visitFile);
+            walk.parallel().forEach(ModDirTransformerDiscoverer::visitFile);
```

Comment:
Walk in parallel to open file systems in parallel, should save some time too based on ATM9 profiles.

Reply:
Does this actually work (i.e. does `Files.walk`'s stream actually support parallelism, or is it a no-op)? Usually I like to collect into a list first and then parallel stream that.

Reply:
Apparently this does not work because of a JDK bug that was only fixed in Java 19: https://bugs.openjdk.org/browse/JDK-8280915. Thanks for noticing...

Reply:
Fixed in 7a0fecd.